### ISSUE TITLE
Red/green color choices not colorblind friendly

### ISSUE THREAD

**Issue by jeffposnick** [PolymerLabs/project-health#374]
## The Problem

The "Ready to Merge" and "Requires Attention" colors are hard to distinguish for users with the type of colorblindness that I have.

https://ux.stackexchange.com/questions/94696/color-palette-for-all-types-of-color-blindness

<img width="318" alt="screen shot 2018-03-29 at 2 20 44 pm" src="https://user-images.githubusercontent.com/1749548/38106255-666793e8-335c-11e8-84c9-6a0465ed3819.png">

## Expected Behavior

Use a red/green pair with more contrast, or an alternative color pair, or a visual cue like bold/italtics to distinguish the two states.

**Comment by merlinnot** [comment]
It actually has a value, since the colours are reflected in actions for the entities:
- No reviewers assigned -> requires attention
- Changes requested -> new activity

Maybe we could use some emojis? 😄
Something like:
- Ready to merge: ✔️
- Requires attention:❗️
- New activity: 🔥 

Developers usually like emojis and they're so lightweight.

**Comment by cdata** [comment]
I agree, the color has meaning and it would actually be more confusing if I could not distinguish red/green. This isn't meant to be a simulation of color blindness, but if I saw a legend that looked like this:

![image](https://user-images.githubusercontent.com/240083/38155918-5c27334c-342f-11e8-8b22-bd1e0329277c.png)

I would be pretty confused.

**Comment by samuelli** [comment]
@cdata per the link in the issue, I think its more like 13/14:

![image](https://user-images.githubusercontent.com/787668/38156411-8bb47f2c-3432-11e8-86d9-4d0be427643c.png)

So I think the issue is more when you're scanning the list it hard to tell which one it is, not that the legend doubles up per se.

**Comment by cdata** [comment]
Certainly, although this comment caught my attention: "The text is designed to be fully sufficient without color." If that's true, then I should be able to desaturate the page and still be able to pick out different categories of content based on the legend.

On a related note: I had no idea that I could filter by clicking on the legend items until you mentioned it in that same comment ;)
